### PR TITLE
enhancement: add transparent background to sponsor images

### PR DESCRIPTION
**Pull request by christian-heusel**
Currently the images sometimes were not using transparent background or were a little pixelated, which this PR tries to fix.

Old:
image

New:
image

### CODE REVIEW

**Review by lcarilla — APPROVED** [review]
lgtm :) 
really looks nicer this way

File: templates/public/donate.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h3>Commercial sponsors and contributions</h3>
     of git commits and software packages, GnuPG keys, and SSH accounts.</p>
 
     <a href="https://nitrokey.com/" title="Nitrokey">
-    <img src="{% static "nitrokey_logo.png" %}"
+    <img width="240" src="{% static "nitrokey_logo.png" %}"
```
Comment (jelly) [inline templates/public/donate.html:53]: Why not resize the original image back to 420px?

Comment (jelly) [reply templates/public/donate.html:53]: This logo is only shown here, it was 240px width, so we can just retain that:

```
magick convert -resize 240 nitrokey_logo.png nitrokey_logo_res.png
```

**Review by jelly — COMMENTED** [review]
Please also oxipng all the images so they are about 10-20% smaller.

**Review by jelly — APPROVED** [review]
oxipng'd the icons, resized the nitrokey logo. All good now.

**Comment by jelly** [comment]
Thanks!

**Pull request merged**